.asf.yaml

```diff
@@ -25,28 +25,29 @@ notifications:
 # https://cwiki.apache.org/confluence/pages/viewpage.action?spaceKey=INFRA&title=Git+-+.asf.yaml+features#Git.asf.yamlfeatures-JenkinsPRwhitelisting
 jenkins:
   github_whitelist:
-    - vcrfxia
     - gharris1727
+    - vcrfxia
     - divijvaidya
-    - lucasbru
```

Comment:
Lucas is now a committer, removing him from the list.

Reply:
I think this is another Lucas. I might be mistaken, but `lucasbru` seems to be Lucas Brutschy, while it is Lucas Bradstreet the one with commit rights.

Reply:
@jlprat is correct. This is a different Lucas.

Reply:
Oops, updated now. Thanks.